src/mappings/helpers.ts

```diff
@@ -111,7 +111,7 @@ export function createOrLoadIndexer(id: string, timestamp: BigInt): Indexer {
     indexer.ownStakeRatio = BigDecimal.fromString('0')
     indexer.delegatedStakeRatio = BigDecimal.fromString('0')
     indexer.delegatorShares = BigInt.fromI32(0)
-    indexer.delegationExchangeRate = BigDecimal.fromString('0')
+    indexer.delegationExchangeRate = BigDecimal.fromString('1')
```

Comment:
im sure this is okay, but just curious, what made you change it to 1?

Reply:
Both for consistency, since `delegationExchangeRate` always starts at 1 when the first delegations flow into the indexer.
Both values are overwritten by their corresponding calculations on stake delegation though, so it's mostly to keep consistency of the values, since they can never be actually 0.